src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx

```diff
@@ -51,10 +64,6 @@ export default function ViewViabilityTestModal(props: ViewViabilityTestModalProp
     onEdit();
   };
 
-  const titleStyle = {
-    color: '#859799',
```

Comment:
you can keep this as-is.

```
import { ..., useTheme } from '@mui/material';
...
const theme = useTheme();
...
const titleStyle = {
  color: theme.palette.TwClrTxtSecondary,
};
```
i believe `sx` is preferred over `makeStyles/className` where feasible - such as here in the previous version.
something to do with mui v5 etc. Constanza knows more about this preferred usage.

Reply:
oh! okay, i felt it was more organized to put everything I could up into `makeStyles`, but if there is a technical reason to prefer `useTheme` and `sx` then I'm happy to change this back and stick with that convention.